frontend/src/components/Login.vue

```diff
@@ -5,18 +5,18 @@ import Button from 'primevue/button';
 import { useAuthStore } from '@/store/authStore';
 
 const authStore = useAuthStore();
-const { ready, getToken } = storeToRefs(useAuthStore());
+const { ready, getKeycloak } = storeToRefs(useAuthStore());
 
 const login = () => {
   authStore.login();
 };
 
-const logout = async () => {
-  const response = await authStore.logout();
+const logout = () => {
+  authStore.logout();
 };
```

Comment:
Do these need to be encapsulated as arrow functions? They may be invokable directly by just doing...
```js
const login = authStore.login();
const logout = authStore.logout();
```
That way it removes the need to have an intermediary anonymous function just to invoke the same thing. Don't know if that collides with Vue3 best practices or not.

Reply:
Yeah that's the good Pinia way of doing it anways, there was some unneeded await there so we can simplify now 👍